.ci/metrics/metrics.py

```diff
@@ -143,9 +302,13 @@ def github_get_metrics(
             break
 
         # This workflow is not interesting to us.
-        if task.name not in GITHUB_WORKFLOW_TO_TRACK:
+        if (task.name not in GITHUB_WORKFLOW_TO_TRACK):
```

Comment:
The extra parantheses seem to be left over from a formatting change. Can you remove them to keep the diff clean?

Reply:
Done.